manual/mailings/call_for_papers.rst

```diff
@@ -15,19 +16,22 @@ scientific computing in Python, such as:
 - hardware systems designed to support the Python stack
 - applications of scientific computing in education
 
-Papers should be submitted as pull requests by May 30th.  At that point, the 
-two-week open review period will begin.  During this time, members from the 
-program committee and the scientific Python community at large may provide 
+Papers must be submitted as pull requests by May 14th.  At that point, the 
+six-week open review period will begin.  During this time, members from the 
+program committee and the scientific Python community at large will provide 
 feedback, which should be addressed through discussion on GitHub and by 
 updating the pull request.
 
+When submitting a paper to the proceedings, you may be contacted by the
+Proceedings Co-chairs to assist in the review process
+
 Further instructions can be found here:
 
 https://github.com/scipy-conference/scipy_proceedings
```

Comment:
We should make sure to update this link if we move the author instructions into their own file. And we should probably make the instructions into their own file so that authors can learn what it takes to submit a paper separate from all the other information about reviews and stuff.